src/style/mixins.scss

```diff
@@ -12,6 +12,16 @@
 }
```

Comment:
I assume only these two last commits are actually new?

<img width="1252" alt="image" src="https://user-images.githubusercontent.com/154725/212638824-69cfa1f3-1667-43c0-99e5-44ea9e0b3c5c.png">

I can help rebase this branch correctly, if you want me to.

Reply:
oh yes. please do. No idea what happened  :/ did I rebase on an old branch?

Reply:
> did I rebase on an old branch?

Something like that probably, yes 😅

No worries though! 🙂

Reply:
Git even solved it automatically when I did an interactive rebase… 🤷‍♂️ 😄